components/Dialogs/src/MDCAlertController.h

```diff
@@ -49,6 +49,9 @@
 /** Alert controllers must be created with alertControllerWithTitle:message: */
 - (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_UNAVAILABLE;
 
+/** Alert controllers must be created with alertControllerWithTitle:message: */
+- (nonnull instancetype)init NS_UNAVAILABLE;
```

Comment:
I don't understand why this is unavailable. The two properties (title, message) are settable below.